modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -526,6 +526,14 @@ impl ClientModule for WalletClientModule {
                         .map_err(|e| anyhow::anyhow!("peg_out failed: {}", e))?;
                     let result = serde_json::to_value(&response)?;
                     yield result;
+                },
+                "check_pegin_status" => {
+                    let req: PeginStatusRequest = serde_json::from_value(request)?;
+                    let response =self.check_pegin_status(req.operation_id)
```

Comment:
Please fix the missing whitespace, `just format` can't reach into macros sometimes.